### PR TITLE
feat: parameterize plugin install command

### PR DESCRIPTION
**Pull request by mstruebing**
Parameterize the install command and remove hardcoded go installation

```
go run main.go -v6 plugin install weed 3.29 https://github.com/mstruebing/asdf-weed.git
go run main.go -v6 plugin install nodejs 16.16.0
```

### CODE REVIEW

File: cmd/plugin/plugin.go [hunk 1]
```diff
@@ -40,17 +40,31 @@ func InstallCommand(out output.Output) *cobra.Command {
...
+			}
+
+			pluginName := args[0]
+			pluginVersion := args[1]
+
+			var pluginURL string
+
+			if len(args) > 2 {
+				pluginURL = args[2]
+			}
```
Comment (mstruebing) [inline cmd/plugin/plugin.go:54]: Is there a better way in go to do this?

Comment (aweris) [reply cmd/plugin/plugin.go:54]: we should use flags instead of positional arguments

Comment (mstruebing) [reply cmd/plugin/plugin.go:54]: I will have a look right now at flags.

Comment (mstruebing) [reply cmd/plugin/plugin.go:56]: I'm using flags in the last commit, not sure if this is the best way to do this.

**Pull request closed without merging**